entitlement/src/test/java/org/killbill/billing/entitlement/api/TestDefaultEntitlementApiFast.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2010-2013 Ning, Inc.
```

Comment:
I 'll fix the copyright, don't want to start another CI run just yet.

Reply:
FYI the feature finally shipped last week:
<img width="261" alt="image" src="https://user-images.githubusercontent.com/51940/159253941-dbdb4468-0520-4901-bd54-f5f32768d74d.png">